### PR TITLE
feat(route): add V2rayShare

### PR DESCRIPTION
**Pull request by 77taibai**
## 该 PR 相关 Issue / Involved Issue

Close #

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/v2rayshare
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/other.md [hunk 1]
```diff
@@ -390,6 +390,16 @@ board 和 build 可在[这里](http://api.ineal.me/tss/status)查看
 
 </Route>
 
+## V2rayShare
+
+### 免费节点
+
+<Route author="钛白" example="/v2rayshare" path="/v2rayshare">
```
Comment (TonyRL) [inline docs/other.md:397]:
```suggestion
<Route author="77taibai" example="/v2rayshare" path="/v2rayshare">
```
author should be your GitHub ID

File: lib/v2/v2rayshare/maintainer.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/': ['钛白'],
```
Comment (TonyRL) [inline lib/v2/v2rayshare/maintainer.js:2]:
```suggestion
    '/': ['77taibai'],
```

File: lib/v2/v2rayshare/index.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+const got = require('@/utils/got'); // 自订的 got
+const cheerio = require('cheerio'); // 可以使用类似 jQuery 的 API HTML 解析器
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { data: response } = await got('https://v2rayshare.com/');
+    const $ = cheerio.load(response);
+
+    const body = $('body > main > div > div > div.content-wrapper.col-lg-8 > div > div'); // 正文列表
```
Comment (TonyRL) [inline lib/v2/v2rayshare/index.js:9]: Please use [WordPress REST API](https://developer.wordpress.org/rest-api/) for a WordPress-based site instead:
- Use `/wp-json/wp/v2/posts` to obtain recent posts

More info: https://github.com/DIYgod/RSSHub/pull/12824#discussion_r1265631483

File: lib/v2/v2rayshare/index.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+    const items = [];
+    for (let i = 0; i < body.length; i++) {
+        const im = body.eq(i).children().eq(0).children().eq(0); // 标题的a标签
+        const year = im.attr('title').split('」')[1].split('年')[0];
+        const month = im.attr('title').split('「')[1].split('月')[0].padStart(2, '0');
+        const day = im.attr('title').split('月')[1].split('日')[0].padStart(2, '0');
+        const html = `<b>Clash节点：</b><a href='https://v2rayshare.com/wp-content/uploads/${year}/${month}/${year}${month}${day}.yaml'>https://v2rayshare.com/wp-content/uploads/${year}/${month}/${year}${month}${day}.yaml</a></br></br>温馨提示：本站提供的都是免费且公共的节点，稳定性与长时间连接速率无法与收费版的高速机场节点相提并论，不能奢望太多。</br>注意你自己的网络环境（本地连接当中的DNS，手动配置一下：114.114.114.114，8.8.8.8或是其它公共的DNS。）</br>如果某个网址无法代理访问，可切换到全局代理模式，一般可解决。`;
```
Comment (TonyRL) [inline lib/v2/v2rayshare/index.js:17]: Please use `art-template` if you want to render custom HTML.
docs: https://docs.rsshub.app/joinus/script-standard.html

File: lib/v2/v2rayshare/index.js (hunk 3 above)
Comment (77taibai) [reply lib/v2/v2rayshare/index.js]: 非常感谢，已经进行修改

File: lib/v2/v2rayshare/index.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        const a = {
+            title: data.title.rendered,
+            link: data.link,
+            author: 'V2rayShare',
+            category: ['免费节点'],
+            pubDate: parseDate(data.date),
+            description: data.content.rendered,
+        };
+        items.push(a);
+    }
```
Comment (TonyRL) [inline lib/v2/v2rayshare/index.js:19]:
```suggestion
    const items = response.map((item) => ({
        title: item.title.rendered,
        link: item.link,
        author: 'V2rayShare',
        category: ['免费节点'],
        pubDate: parseDate(item.date_gmt),
        description: item.content.rendered,
    }));
```
You can use [.map()](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create a new array.

Comment (77taibai) [reply lib/v2/v2rayshare/index.js]: 非常感谢~

**Pull request merged**